test/functional/interface_zmq.py

```diff
@@ -455,10 +452,10 @@ def test_mempool_sync(self):
 
         # Some transactions have been happening but we aren't consuming zmq notifications yet
         # or we lost a ZMQ message somehow and want to start over
-        txids = []
+        txs = []
```

Comment:
Unrelated: It looks like this array is unused beside the last value? Could remove it and just assign the last value to a non-array name?

Reply:
agreed, was very confused by this array. ended up leaving it alone just to minimize my changes to just `MiniWallet`

Reply:
Yes, it is good to keep unrelated changes for separate commits and/or follow-ups.